django/core/servers/basehttp.py

```diff
@@ -77,6 +77,10 @@ def handle_error(self, request, client_address):
             super(WSGIServer, self).handle_error(request, client_address)
 
 
+class ThreadedWSGIServer(socketserver.ThreadingMixIn, WSGIServer):
+    """ A threaded version of the WSGIServer """
```

Comment:
Please remove the leading trailing space to be consistent with other docstrings. I'm not sure if there are other classes that are empty without using `pass` (`django/core/exceptions.py` comes to mind). I'm not sure if there's any style guide for doing it one way or another.